src/lib/input/input-container.html

```diff
@@ -30,6 +36,6 @@
           [class.md-warn]="dividerColor == 'warn'"></span>
   </div>
 
-  <div *ngIf="hintLabel != ''" [attr.id]="_hintLabelId" class="md-hint">{{hintLabel}}</div>
+  <div *ngIf="hintLabel != ''" [attr.id]="_hintLabelId" class="md-hint, mat-hint">{{hintLabel}}</div>
```

Comment:
Oh bugger I totally put that in the wrong spot, it should have been in the select attribute of the ng-content below it